contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -225,6 +227,10 @@ func NewSchedulerServer() *SchedulerServer {
 		// when kube-proxy is running in a non-root netns (init_net); setting this to a non-zero value will
 		// impact connection tracking for the entire host on which kube-proxy is running. xref (k8s#19182)
 		conntrackMax: 0,
+
+		// cadvisor housekeeping related config
+		housekeepingInterval:       1 * time.Second,
```

Comment:
I am very puzzled on where the housekeeping_interval=10s is specified...  I searched the cadvisor code, only see 1s is specified.   it seems we should remove it from 'km scheduler --help', because we don't support passing it via the housekeeping_interval with "_" in it (only dash is supported).